_data/events.yaml

```
@@ -2,13 +2,37 @@ headline: "Select Upcoming Events"
 
 upcomingevents:
 
+- title: "Voxxed Days - Ticino"
+  type: In Person
+  link: https://voxxeddays.com/ticino/
+  date: "January 20, 2023"
+  location: Ticino, Switzerland
+  description: VOXXED Days Ticino is a 1-day event dedicated to open source (Java, web, mobile and languages.
+  thumbnail: voxxeddays_ticino.png
+
+- title: "JFokus"
```

```suggestion
- title: "Jfokus"
```
(I checked on the website as the title and the description were inconsistent)